pyproject.toml

```diff
@@ -75,6 +78,7 @@ docs = [
     "myst-parser",
     "sphinx-autobuild",
 ]
+s3fs = ["fs-s3fs"]
```

Comment:
As an 'extras' name, I might slightly prefer the shorter and more intuitive `s3`. What do you think?

Reply:
I went ahead and updated the 'extras' name to `s3`. I think it does make more sense that way!